rails/spec/requests/api/public_stories_spec.rb

```diff
@@ -180,5 +215,16 @@
       expect(json_response["stories"].count).to eq(1)
       expect(json_meta["hasNextPage"]).to be false
     end
+
+    it "orphaned stories are excluded" do
+       # ensures that story_with_no_speaker does not show up in list:
+      speaker_being_deleted.destroy
+      # ensures that story_with_no_place does not show up in list:
+      place_being_deleted.destroy
+
+      get "/api/communities/orphaned_community/stories"
+      expect(json_response["total"]).to eq(0)
+      expect(json_response["stories"].length).to eq(0)
+    end
```

Comment:
I'd like to verify if this test is necessary as I believe there are specs relating to this already (see lines 165/175 in this file). If you remove the code in the `api/stories_controller.rb` and rerun this test (after adjusting the `json_response` -> `json_meta` for `total`, does it still pass? If so, let's remove it.

Reply:
Another unfortunate case of code being changed after the PR was created. Looks like the tests in lines 165/175 were created by you about a week after my PR was opened. I'll delete this test and the speaker and place being deleted.
For reference, I had originally used some of the old stories, but the first request for change had me create new ones.

Reply:
Ah, my apologies for the run around.